lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -76,6 +77,10 @@ def setUp(self):
             self.client = Client()
             assert_true(self.client.login(username=uname, password=password))
 
+        config = ForumsConfig.current()
+        config.enabled = True
+        config.save()
```

Comment:
Is it worth making a base class for all of these test classes that turns forums on?